jxbrowser/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>jxbrowser</artifactId>
+	<packaging>eclipse-plugin</packaging>
+
+	<parent>
+		<groupId>ch.sbb.scion</groupId>
+		<artifactId>scion-parent</artifactId>
+		<version>0.0.9-SNAPSHOT</version>
+	</parent>
+
+	<name>JxBrowser Core API and SWT integration</name>
+
```

Comment:
Same as above.

Reply:
@armiol Could you please clarify?

Reply:
@Dmytro-Fedoryshyn I had another comment, probably, related to [this line](https://github.com/TeamDev-IP/scion-rcp/pull/1/files#diff-feaf0961595af640058ad0e2e1f50398cac723561341dfe2cf11f3375b55e074R18). Not sure where it went.

Let's not double any of empty lines, especially, in a well-structured and well-indented files such as XMLs.